src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java

```diff
@@ -0,0 +1,160 @@
+        Model expectedModel = new ModelManager(model.getResumeBook(), new UserPrefs());
+        expectedModel.setProjectToDisplay();
+        CommandResult expectedCommandResult = new CommandResult("",
+                String.format(Messages.MESSAGE_ITEMS_LISTED, 0, "Projects"),
+                expectedModel.getDisplayType());
+        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+        FindProjectCommand command = new FindProjectCommand(predicate);
+        model.updateFilteredItemList(predicate);
+        assertCommandSuccess(command, model, expectedCommandResult, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredItemList());
+    }
+
+    @Test
+    public void execute_singleKeyword_singleProjectFound() {
+        Model expectedModel = new ModelManager(model.getResumeBook(), new UserPrefs());
+        expectedModel.setProjectToDisplay();
+        CommandResult expectedCommandResult = new CommandResult("",
+                String.format(Messages.MESSAGE_ITEMS_LISTED, 1, "Projects"),
+                expectedModel.getDisplayType());
+        NameContainsKeywordsPredicate predicate = preparePredicate("Orbital");
+        FindProjectCommand command = new FindProjectCommand(predicate);
+        model.updateFilteredItemList(predicate);
+        assertCommandSuccess(command, model, expectedCommandResult, expectedModel);
+        assertEquals(Arrays.asList(ORBITAL), model.getFilteredItemList());
+    }
+
+    @Test
+    public void execute_zeroKeywords_noSkillFound() {
+        Model expectedModel = new ModelManager(model.getResumeBook(), new UserPrefs());
+        expectedModel.setSkillToDisplay();
+        CommandResult expectedCommandResult = new CommandResult("",
+                String.format(Messages.MESSAGE_ITEMS_LISTED, 0, "Skills"),
+                expectedModel.getDisplayType());
+        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+        FindSkillCommand command = new FindSkillCommand(predicate);
+        model.updateFilteredItemList(predicate);
+        assertCommandSuccess(command, model, expectedCommandResult, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredItemList());
+    }
+
+    @Test
+    public void execute_singleKeyword_singleSkillFound() {
+        Model expectedModel = new ModelManager(model.getResumeBook(), new UserPrefs());
+        expectedModel.setSkillToDisplay();
+        CommandResult expectedCommandResult = new CommandResult("",
+                String.format(Messages.MESSAGE_ITEMS_LISTED, 1, "Skills"),
+                expectedModel.getDisplayType());
+        NameContainsKeywordsPredicate predicate = preparePredicate("React");
+        FindSkillCommand command = new FindSkillCommand(predicate);
+        model.updateFilteredItemList(predicate);
+        assertCommandSuccess(command, model, expectedCommandResult, expectedModel);
+        assertEquals(Arrays.asList(REACT), model.getFilteredItemList());
+    }
+
```

Comment:
I think we should add FindResumeTest too?

Reply:
yupp! will add soooon